boards/main/src/main.rs

```diff
@@ -199,29 +198,6 @@ mod app {
         });
     }
 
-    /**
```

Comment:
This doesn't necessarily need to be removed, but the feature for reading/sending the state of the machine is not fully implemented. The message definition needs to change before this is implemented.